scripts/pages.mjs

```diff
@@ -43,6 +43,69 @@ class ClassPages extends Application {
     html[0].querySelectorAll(".class-nav img[data-action]").forEach(n => {
       n.addEventListener("click", this._onClickClass.bind(this));
     });
+    this.navScroller(html);
+  }
+
+  /**
+   * Activates the scroll functionality for the class icons gallery.
+   * @param {JQuery} html The HTML content of the rendered ClassPages application.
+   */
+  navScroller(html) {
```

Comment:
Unless meant to be part of an allowed API, functions like these should be underscored.